8.2.13 Release # DO NOT DELETE

su-sws-jira opened the pull request.

su-sws-jira committed:
8.2.13

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

su-sws-jira committed:
Updated dependencies Jun 24 2020

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

Commit:
feat(STN-223): unique menu IDs for secondary nav menus (#632)

kaseybon committed:
feat(STN-325): Private Pages and Utility (#636)

* feat: add a lock icon to private page titles, menu links, and with a utility class
* chore: add compiled CSS

kaseybon committed:
fix(STN-439): Heading Link Hover Color (#641)

* fix: links in headings should be black on hover
* chore: commit compiled CSS

kaseybon committed:
fix(STN-406): IE 11 Text Overflow in Rows (#640)

* fix: ie 11 text was overflowing the width of the page when a single card (postcard) was used in a row

* fix: heading link text color on focus should be black (should have gone in another PR, #641  but we didn't realize at the time)

* chore: commit compiled CSS

Commit:
feat(STN-417):  QA Feedback for Sprint 18 (#639)

* feat(417): accordion fix

* fix(417): accordion not in well row"

* fix(STN-417): custom content type hb-categories

* fix(STN-417): long text select issue on iphone 11

* feat(STN-417): finishing up date card fixes

* refactor(STN-417): remove unnecessary code

* feat(STN-417): add hb-single-category utility class

STN-417

* chore(STN-417): compiled CSS

Co-authored-by: Kasey Bonifacio <kasey@heysparkbox.com>

Commit:
feat(STN-419): Spotlight Paragraph Type and UI Pattern (#633)

* feat(STN-419): New Paragraph Type and Pattern HS Spotlight

* feat(STN-419): css changes for spotlight

* feat(STN-419): image media entity type display change

* refactor(STN-419): slight design tweak to headings

* fix(STN-419): behat test

* refactor(STN-419): sass

* feat(STN-419): Spotlight fields for display and image_alignment

- also adds CSS for image_aligment
- adds config export for Spotlight paragraph type changes to fields, and
  display settings

Commit:
chore: compiled CSS

pookmish committed:
HSD8-844 Dont trim the link field for vertical linked cards

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

su-sws-jira committed:
Updated dependencies Jul 1 2020

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

Commit:
feat: adds Spotlight UI pattern to SU humsci theme

pookmish committed:
Sparkbox Sprint 19

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

Commit:
fix: private page links, icons on custom menus, and private page textarea (#644)

* fix: private page icons on custom menus and private page textarea

- some important preprocess functions werent running on any other menus
  other than the main menu, now they will run on everything, including
custom menus so we can get the appropriate class applied
- additionally, the private textarea was missing certain textstyles, so
  that has been remedied too

* fix: allow for the private link to work on fields in node pages (#643)

* fix: allow for the private link to work on fields in node pages

* fix: prevent the colorful link gradient hover from sitting behind the lock icon

* chore: commit css

* refactor: simplify link hover scss

Co-authored-by: Kasey Bonifacio <kasey@heysparkbox.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

pookmish committed:
HSD8-843 Remove subscription information from the toolbar

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

pookmish committed:
HSD8-840 HSD8-841 Provision humanbiology and mediterraneanstudies

joegl committed:
HSD8-825: Added event_series to event importer (#645)

* Added event_series to event importer

* Removed event series id

* Added field_hs_event_series to overwrite_properties

* Reverted overwrite_properties

Co-authored-by: pookmish <pookmish@gmail.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

pookmish committed:
HSD8-845 Add contextual links for blocks placed via layout builder

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

pookmish committed:
HSD8-849 Change media libary to a normal pager

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

pookmish committed:
HSD8-833 Fixed permissions for media overview and toolbar links

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

Commit:
fix: revert Spotlight paragraph type for now (#647)

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

pookmish committed:
HSD8-835 Adjustments to the publication importer documents and citation field

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

pookmish committed:
fixed behat tests

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr638-rc8r7cuzskqtlrjm6ey6l2rdy7msozqb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837

Visual Diffs:
- https://dashboard.tugboat.qa/5ef2a803f6b4839b7e24a837/visualdiffs

pookmish committed:
HSD8-848 Re-enable shield on dev and stage sites
